lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -422,23 +440,11 @@ void SessionStateEnforcer::set_new_fsm_state_and_increment_version(
 }
 
 void SessionStateEnforcer::m5g_pdr_rules_change_and_update_upf(
-    const std::string& imsi, const std::unique_ptr<SessionState>& session,
-    PdrState pdrstate) {
+    const std::unique_ptr<SessionState>& session, PdrState pdrstate) {

```

Comment:
```suggestion
    const std::unique_ptr<SessionState>& session, PdrState pdr_state) {

```

Reply:
renamed the parameter from pdrstate to pdr_state as suggested